modules/openapi-generator/src/main/resources/Java/libraries/resttemplate/api.mustache

```diff
@@ -13,6 +13,11 @@ import java.util.Locale;
 import java.util.Map;
 import java.util.stream.Collectors;
 
+{{#useBeanValidation}}
+import {{javaxPackage}}.validation.constraints.*;
```

Comment:
The pom.xml generated for resttemplate always includes the `jakarta` dependency and never the `javax` dependency. But `{{javaxPackage}}` used in this line by default refers to javax. So please either change the pom.xml or hard-code jakarta here.